en_US/advanced/cluster.md

```diff
@@ -341,3 +341,53 @@ also be allowed by the firewall. The port mapping rule is similar to the node di
 ports in `ekka mode`, but with the `BasePort = 5370`. That is, having
 `node.name = emqx@192.168.0.12` in `emqx.conf` should make the node listen on port `5370`,
 and port `5371` for `emqx1` (or `emqx-1`), and so on.
+
+## Asynchronous database replication
+
+::: danger
+This feature is highly experimental.
+You can enable it if you want to help us testing and tuning it.
+:::
+
+Some EMQ X features rely on the open-source [Mnesia](http://erlang.org/doc/apps/mnesia/Mnesia_overview.html) database.
+Mnesia uses full-mesh topology for transaction replication.
+While being blazing-fast in the small clusters, it has some scalability issues when the number of nodes in the cluster becomes large.
+Also it becomes more vulnerable to network splits.
+To address these issues, EMQ X supports an experimental extension to Mnesia called `RLOG`.
+This extension can be used in large EMQ X clusters (4 nodes and more).
+
+### RLOG principles
+
+RLOG extension works by separating nodes in the cluster into two logical groups:
+
+- core nodes
+- replicate nodes
+
+Core node are the source of truth for the database.
+They contain an up-to-date consistent replicas of the tables.
+It's better to have several core nodes in the cluster to increase redundancy of the data.
+Note: it is not recommended to put core nodes into auto-scaling groups.
+
+Replicant nodes passively and asynchronously replicate data from the core nodes.
+They can handle MQTT traffic, but they delegate transaction coordination to the core nodes.
+It is possible to put replicate nodes into an auto-scaling group.
+
+::: warning
+It takes time for the new replicant to join the cluster, and this process generates some additional load on the core nodes.
+Don't use too aggressive autoscaling settings to avoid frequent replicant restarts.
+:::
+
+Flow of the data in the cluster is illustrated in the following diagram:
+
+![image](./assets/rlog_cluster.png)
+
+### Enabling RLOG feature
+
+Let's suppose we delegate `core` role to nodes `emqx@core1.internal` and `emqx@core2.internal`.
+
+The following changes should be done in the `etc/emqx.conf` file:
+
+1. Set `cluster.db_backend` parameter to `rlog` on all nodes in the cluster
```

Comment:
is it for provsioing new cluster?

clarivify if migration is supported or not.

Reply:
Added a bullet for stopping the cluster.